node-engine/guides/create-app.mdx

```diff
@@ -4,16 +4,18 @@ title: Create Your First App
 description: The best way to understand how a Headless CMS operates is to create a simple application and deploy it. Use WPEngine's headless framework to create a simple application.
 ---
 
-The best way to understand how a Headless CMS operates is to create a simple application and deploy it. This tutorial uses WPEngine's open-source [headless framework](https://github.com/wpengine/faustjs) and Next.js to create a simple application.
+The best way to understand how a Headless CMS operates is to create a simple application and deploy it. This tutorial uses Faust.js, WPEngine's open-source [headless framework](https://github.com/wpengine/faustjs), and Next.js to create a simple application.
 
 ## Setup WordPress
 
 1. Make a WordPress site. You should have a site and an Atlas account through WPEngine in the [User Portal](https://my.wpengine.com). Navigate to the site environment you want to use for your backend, and click the WP Admin link at the top of the page. This will log you into WP Admin, where the setup for the framework plugin and WPGraphQL occurs.
-1. Download, upload, and activate the `wpe-headless` plugin. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
-1. Install [WP GraphQL](https://wordpress.org/plugins/wp-graphql/) on the WordPress site. You can automatically install WPGraphQL from the framework plugin's **settings** page
+1. Download the [`wpe-headless` plugin](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download). Then, upload and activate the plugin from within the WordPress admin. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
```

Comment:
- Remove the ` marks around "wpe-headless".

- Same note as before about WordPress admin.

- Remove the "[Plugin Download]" link at the end of this step.